tests/test_docker_utils.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from girder_worker.docker.utils import ( # noqa F401
+    select_loop,
+    chmod_writable
+)
+
+
+# TODO: test select_loop
+@pytest.mark.skip
+def test_select_loop():
+    pass
+
+
+# TODO; test chmod_writable
+@pytest.mark.skip
+def test_chmod_writable():
+    pass
```

Comment:
I think we should remove these. Not sure there would be much value in the test as they would be pretty mocked out. They are pretty will covered by the integration tests.